virttest/utils_hotplug.py

```diff
@@ -26,6 +27,7 @@
 from . import utils_misc
 from . import utils_test
 from .utils_test import libvirt
+from .libvirt_xml.devices import memory
```

Comment:
request to avoid relative imports,
`from virttest.libvirt_xml.devices import memory`
better to change other imports too.

Reply:
replaced all relative reference to absolute references.